src/components/search/omnibox-directions.component.tsx

```diff
@@ -122,7 +123,7 @@ const OmniboxDirections = ({
     if (currentLocation && !startLocation) {
       setStartLocation({
         id: CURRENT_LOCATION_DISPLAY_TEXT,
-        displayName: CURRENT_LOCATION_DISPLAY_TEXT,
+        displayName: startLocationDisplay,
```

Comment:
Why was this changed? I feel like it might break something, especially with the new changes merged recently by  #101 and #102. Lines 104 to 120 already take care of syncing up the display text whenever either the start or end locations are updated.

Reply:
Nevermind I see why this was done. But it shouldn't be done like this. 

Please revert this and follow the comment tagged FIX-NOTE-1

Reply:
Here is why this would be an issue:
![image](https://user-images.githubusercontent.com/23544999/79019608-bb625e00-7b44-11ea-803a-ad93acc26899.png)

Notice that the start location isn't set quite correctly, which could lead to issues in the future.

Also, you remarked that you were getting issues with the latest merge #102   with "Fetching Current Location ... " staying and not changing. But that issue is caused by your changes.

Reply:
Issue is addressed.